SECURITY.md

```diff
@@ -0,0 +1,13 @@
+# Security Policy
+
+## Supported Versions
+
+[Currently supported versions](https://docs.janusgraph.org/changelog/#currently-supported) of JanusGraph
+continue to receive security updates. If you are using an older version of JanusGraph that is no longer
+supported, please update to a supported version to continue to receive security updates.
+
+## Reporting a Vulnerability
+
+Please security vulnerabilities privately to the JanusGraph Technical Steering Committee (TSC) by sending
```

Comment:
I'm so sorry, looks like I missed the word "report" in there; it should be:

> Please report security vulnerabilities [...]

Reply:
Fixed. Thanks!